doc/classes/ResourceImporterBMFont.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="ResourceImporterBMFont" inherits="ResourceImporter" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+	</brief_description>
+	<description>
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="compress" type="bool" setter="" getter="" default="null">
+		</member>
```

Comment:
Default values don't seem to be included properly, yet they're seen as "valid" so it shows `default="null"` for all (unlike e.g. EditorSettings where it doesn't show `default`).

Reply:
I pushed a second commit that solves this by calling `get_import_options` directly from DocTools so we can have access to the default value in `ImportOption`. It's a bit more hacky though. 09f0f70d81f50f7740268bcba4aff2106a9ec8cd

The less hacky option would be to properly register import options as properties in ClassDB with their default values, and then we don't need any of these hacks.